common/hardware_config.go

```diff
@@ -6,14 +6,16 @@ import (
 )
 
 type HardwareConfig struct {
-	CPUs           int32 `mapstructure:"CPUs"`
-	CPUReservation int64 `mapstructure:"CPU_reservation"`
-	CPULimit       int64 `mapstructure:"CPU_limit"`
-	RAM            int64 `mapstructure:"RAM"`
-	RAMReservation int64 `mapstructure:"RAM_reservation"`
-	RAMReserveAll  bool  `mapstructure:"RAM_reserve_all"`
-	DiskSize       int64 `mapstructure:"disk_size"`
-	NestedHV       bool  `mapstructure:"NestedHV"`
+	CPUs                int32 `mapstructure:"CPUs"`
+	CPUReservation      int64 `mapstructure:"CPU_reservation"`
+	CPULimit            int64 `mapstructure:"CPU_limit"`
+	RAM                 int64 `mapstructure:"RAM"`
+	RAMReservation      int64 `mapstructure:"RAM_reservation"`
+	RAMReserveAll       bool  `mapstructure:"RAM_reserve_all"`
+	DiskSize            int64 `mapstructure:"disk_size"`
+	NestedHV            bool  `mapstructure:"NestedHV"`
+	CpuHotAddEnabled    bool  `mapstructure:"cpu_hot_add_enable"`
+	MemoryHotAddEnabled bool  `mapstructure:"memory_hot_add_enable"`
```

Comment:
what do you think if we name them `CPU_hot_plug` and `RAM_hot_plug` to match vSphere UI language?

Reply:
Fixed